go/validationlib/input_gen.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
```

Comment:
As a first pass, let's just stick with manually written table tests in unit tests. We can add this fuzzing later on.

Also unit tests should not be using random inputs like this, as it can make it non-deterministic.

Reply:
some test would require input string with length > 5000.
It may look strange to put a very long string in test? 
The goal of the generator was mainly to avoid such situation.

Reply:
done,